#// Структура для углов наклонов

	float32 roll          # Крен в право  влево
	float32 pitch         # Тангаж вверх или вних
	float32 yaw           # Поворот по часовой мом против часовой
	float32 x             # Координата по Х
	float32 y             # Координата по Y
	float32 th            # Направление носа
	float32 vel_x         # Линейная скорость движения робота по оси X
	float32 vel_y         # Линейная скорость движения робота по оси Y
	float32 vel_th        # Угловая скорость вращения робота